examples/tree_1d_dgsem/elixir_mhd_torrilhon_shock_tube.jl

```diff
@@ -7,6 +7,26 @@ using Trixi
 gamma = 5/3
 equations = IdealGlmMhdEquations1D(gamma)
 
+"""
+    initial_condition_torrilhon_shock_tube(x, t, equations::IdealGlmMhdEquations1D)
+
+Torrilhon's shock tube test case for one dimensional ideal MHD equations.
+- Torrilhon (2003)
+  Uniqueness conditions for Riemann problems of ideal magnetohydrodynamics
+  [DOI: 10.1017/S0022377803002186](https://doi.org/10.1017/S0022377803002186)
+"""
+function initial_condition_torrilhon_shock_tube(x, t, equations::IdealGlmMhdEquations1D)
+  # domain must be set to [-1, 1.5], γ = 5/3, final time = 0.4
+  rho = x[1] <= 0 ? 3.0 : 1.0
+  v1 = 0.0
+  v2 = 0.0
+  v3 = 0.0
+  p = x[1] <= 0 ? 3.0 : 1.0
+  B1 = 1.5
+  B2 = x[1] <= 0 ? 1.0 : cos(1.5)
+  B3 = x[1] <= 0 ? 0.0 : sin(1.5)
```

Comment:
Same question on <= vs <

Reply:
@andrewwinters5000? You added this in 518899dbc13592006cd25b25c49c5cac1f8b9809